### ISSUE TITLE
 How do I record calculated scores using target variables before conversion? (e.g. log conversion)

### ISSUE THREAD

**Issue by wakamezake** [nyanp/nyaggle#67]
This code is recorded the score of the target variable transformed by `np.log1p`.
```python
train, test = load_dataset()
target_col = "y"
submit = make_sample_submission(test, target_col)
target = train[target_col]
target = target.map(np.log1p)
train.drop(columns=[target_col], inplace=True)
lightgbm_params = {
        "metric": "rmse",
        "objective": 'regression',
        "max_depth": 5,
        "num_leaves": 24,
        "learning_rate": 0.007,
        "n_estimators": 30000,
        "min_child_samples": 80,
        "subsample": 0.8,
        "colsample_bytree": 1,
        "reg_alpha": 0,
        "reg_lambda": 0,
    }

fit_params = {
        "early_stopping_rounds": 100,
        "verbose": 5000
    }

kf = KFold(n_splits=4)
lgb_result = run_experiment(lightgbm_params,
                                X_train=train,
                                y=target,
                                X_test=test,
                                eval_func=rmse,
                                cv=kf,
                                fit_params=fit_params,
                                logging_directory='resources/logs/'
                                                  'lightgbm/{time}',
                                sample_submission=submit)
```

**Comment by nyanp** [comment]
One possible idea is that recording the metric outside of `run_experiment`.

```python
with Experiment(logging_directory='resources/logs/lightgbm/{time}') as e:
    lgb_result = run_experiment(lightgbm_params,
                                    X_train=train,
                                    y=target,
                                    X_test=test,
                                    eval_func=rmse,
                                    cv=kf,
                                    fit_params=fit_params,
                                    sample_submission=submit,
                                    inherit_experiment=e)
    e.log_metrics("score", ...)
```

ref:
https://nyaggle.readthedocs.io/en/latest/source/tutorial/experiment_advanced.html#logging-extra-parameters-to-run-experiment

One drawback is that the submission file and the test prediction will be calculated based on the transformed data. Another possible solution might be using custom objective / metric function represents RMSLE instead of transforming target.

**Comment by wakamezake** [comment]
Thanks! I'll try it.
